### PR TITLE
Change the background color of the text on oleville list view

### PR DESCRIPTION
**Pull request by elijahverdoorn**
The gold text was hard to read.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "react-native-animatable": "0.6.1",
     "react-native-button": "1.6.0",
     "react-native-calendar": "0.6.4",
+    "react-native-cli": "1.0.0",
```
Comment (hawkrives) [inline package.json]: What is this and why did you add it to the package?

Comment (elijahverdoorn) [inline package.json]: This is in response to the issues that I was having building when I pulled from master. This is the windows CLI implementation that you need to build with the command `react-native run-android`. (I think)

**Comment by hawkrives** [comment]
`react-native` itself comes with the cli implementation. That was merged into the main `react-native` package a while ago.

Please remove it?

**Comment by hawkrives** [comment]
Regardless of whether or not we need it, it doesn't really belong in this PR 😃

**Comment by elijahverdoorn** [comment]
Fair. Removing.

**Comment by hawkrives** [comment]
LGTM! :shipit:
